ui/timer.qml

```diff
@@ -171,8 +173,14 @@ Mycroft.Delegate {
                     anchors.topMargin: Kirigami.Units.largeSpacing
                     color: "#2C3E50"
                     font.family: "Noto Sans"
-                    font.pixelSize: parent.height * 0.15
+                    horizontalAlignment: Text.AlignHCenter
+                    width: timerBackground.width - (Kirigami.Units.largeSpacing + Kirigami.Units.smallSpacing)
+                    height: parent.height * 0.35
+                    font.pixelSize: contentWidth > (width * 0.8) ? timerName.height * 0.4 : timerName.height * 0.6
```

Comment:
The conditional here doesn't seem to be working for me. "Timer 2" is getting truncated (...)

Seems to work well checking against the parent.width eg 
```
contentWidth > (parent.width * 0.8)
```

Will that work on all platforms?